data/categories.txt

```diff
@@ -44,3 +44,5 @@ category "outfit"
 	"Hand to Hand"
 	"Unique"
 	"Special"
+	"Minerals"
+	"Licenses"
```

Comment:
Ordering by how important it is that the player keeps track of these outfits:

- "Unique": These outfits are basically regular outfits, but with limited availability.
- "Minerals": Outfits that are sold. Given that the player will more regularly interact with these outfits than Special outfits, I feel it belongs higher than it.
- "Special": Outfits that stay with the player. They don't really need to keep track of them.
- "Licenses": Glorified conditions.

```suggestion
	"Unique"
	"Minerals"
	"Special"
	"Licenses"
```